### PR TITLE
Replace inquirer with enquirer

### PR DESCRIPTION
**Pull request by vcalasans**
This PR replaces the `inquirer` dependency with `enquirer` as an attempt to solve the freeze in output reported by some Windows users.

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.
- [x] Refactor

### CODE REVIEW

File: src/modules/init/index.ts [hunk 1]
```diff
@@ -88,7 +92,7 @@ const promptTemplates = async (): Promise<string> => {
 }
 
 const promptContinue = async (repoName: string) => {
-  const proceed = prop('proceed', await inquirer.prompt({
+  const proceed = prop('proceed', await enquirer.prompt({
```
Comment (lbebber) [inline src/modules/init/index.ts]: why are you using `enquirer.prompt` here instead of using `promptConfirm`?

Comment (lbebber) [reply src/modules/init/index.ts]: same applies to other similar lines

Comment (vcalasans) [reply src/modules/init/index.ts]: because these are not confirmation prompts!

Comment (vcalasans) [reply src/modules/init/index.ts]: Sry, this was is indeed a `confirm` prompt! Will fix it.

**Pull request merged**